praw/objector.py

```diff
@@ -1,10 +1,29 @@
 """Provides the Objector class."""
+import re
+
 from .exceptions import APIException
 
 
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def camel_to_snake(name):
```

Comment:
Can we make this method private (prefix with `_`)? It's useful, but I don't want to commit to making it part of the public interface.